terminal.go

```diff
@@ -1233,6 +1248,46 @@ func (g *LightningTerminal) BuildWalletConfig(ctx context.Context,
 	)
 }
 
+// buildAuxComponent builds the auxiliary components required by lnd when
+// running in integrated mode with tapd being the service that provides the
+// aux component implementations.
+func (g *LightningTerminal) buildAuxComponents() (*lnd.AuxComponents, error) {
+	tapdWrapper, available := g.subServerMgr.GetServer(subservers.TAP)
+	if !available {
+		return nil, fmt.Errorf("tapd is not available, must be " +
+			"started in integrated mode for Taproot Assets " +
```

Comment:
`started in integrated mode` -> do you mean that tapd must be integrated? cause 'integrated mode' to me means that lnd is integrated

Reply:
Actually, both. I'll update the error message.